jck/build.xml

```diff
@@ -119,41 +119,47 @@
 		<if>
 			<available file="${JCK_ROOT_USED}" type="dir" />
 			<then>
-				<!--Obtain local and remote SHA to figure out if local materials are up-to-date--> 
-				<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="false" outputproperty="localSHA" resultproperty="return.code">
-					<arg line="rev-parse"/>
-					<arg line="HEAD"/>
-				</exec>
 				<if>
-					<not>
-						<equals arg1="${return.code}" arg2="0"/>
-					</not>
+					<isset property="env.SKIP_JCK_GIT_UPDATE" />
```

Comment:
Since SKIP_JCK_GIT_UPDATE is a boolean, the check will always return true as long as it is set. Maybe check to assess the specific value of env.SKIP_JCK_GIT_UPDATE instead of checking if it's set. <equal arg1="{env.SKIP_JCK_GIT_UPDATE}" arg2="true">

Reply:
The original code was just checking the "is set" . I have absolutely no objections to change it  to explicit value. TY!